packages/babel-helpers/src/helpers/applyDecs2303.js

```diff
@@ -537,7 +519,7 @@ function applyClassDecs(targetClass, classDecs) {
     [
       // member decorators
       [
-        dec,                // dec or array of decs
+        decs,               // array of decs or of decs and this values
```

Comment:
Are they always "array of decorator and this values" now?

Reply:
Now that the flag is stored together with the other flags, we can optimize the array away again when we have a single decorator without `this` :)